app/src/main/java/swati4star/createpdf/activity/ImageEditor.java

```diff
@@ -81,7 +85,8 @@ protected void onCreate(Bundle savedInstanceState) {
         ButterKnife.bind(this);
 
         initValues();
-
+        mWatermarkutils = new WatermarkUtils(this);
```

Comment:
There should be no use of watermark here.

Reply:
Okay

Reply:
We dont need watermarkutils now. Remove it & corresponding imports too